docs/queries/bylabeltext.mdx

```diff
@@ -71,14 +71,14 @@ import { render, screen } from '@testing-library/react'
 
 render(<Login />)
 
-const inputNode = screen.getByLabelText('username')
+const inputNode = screen.getByLabelText('Username')
```

Comment:
Can you update the examples so they are all capitalized the same way?

`L53 <input aria-label="username" />`